res/values/strings.xml

```diff
@@ -465,11 +465,10 @@
     <!-- share and forward messages -->
     <!-- Translators: Title shown above a chat/contact list; the user selects the recipient of the messages he wants to forward to -->
     <string name="forward_to">Forward to…</string>
-    <string name="share_multiple_attachments">Send %1$d files to the selected chat?\n\nThe files are sent unmodified in their original size, eg. images and videos are not recoded.</string>
-    <string name="share_multiple_attachments_multiple_chats">Send %1$d file(s) to %2$d chats?\n\nThe files are sent unmodified in their original size, eg. images and videos are not recoded.</string>
+    <string name="share_multiple_attachments">Send %1$d file(s) unmodified to the selected chat?</string>
+    <string name="share_multiple_attachments_multiple_chats">Send %1$d unmodified file(s) to %2$d chats?</string>
     <string name="share_text_multiple_chats">Send this text to %1$d chats?\n\n\"%2$s\"</string>
-    <string name="share_abort">Sharing aborted due to missing permissions.</string>
-
+    <string name="share_abort">Grant all required permissions to share this.</string>
```

Comment:
not sure if that is better error message, however, also not 100% sure where the string appears at all.

Reply:
I like active hints better.